src/components/railBottom/miniPaperControl/miniPaperControl.scss

```diff
@@ -26,6 +26,7 @@
 
     & > svg {
       fill: #333344;
+      width: 100%;
```

Comment:
the minimap svg was looking weird on Electron browser. Making it 100% solved the issue